src/EditorFeatures/Core/Implementation/Diagnostics/DiagnosticsSuggestionTaggerProvider.cs

```diff
@@ -37,16 +39,20 @@ internal partial class DiagnosticsSuggestionTaggerProvider :
             IThreadingContext threadingContext,
             IDiagnosticService diagnosticService,
             IForegroundNotificationService notificationService,
-            IAsynchronousOperationListenerProvider listenerProvider)
+            IAsynchronousOperationListenerProvider listenerProvider,
+            PrimaryWorkspace primaryWorkspace)
             : base(threadingContext, diagnosticService, notificationService, listenerProvider)
         {
+            _primaryWorkspace = primaryWorkspace;
         }
 
         protected internal override bool IncludeDiagnostic(DiagnosticData diagnostic)
             => diagnostic.Severity == DiagnosticSeverity.Info;
 
-        protected override IErrorTag CreateTag(DiagnosticData diagnostic) =>
-            new ErrorTag(PredefinedErrorTypeNames.HintedSuggestion, diagnostic.Message);
+        protected override IErrorTag CreateTag(DiagnosticData diagnostic)
+            => new ErrorTag(
+                PredefinedErrorTypeNames.HintedSuggestion,
+                CreateToolTipContent(_primaryWorkspace.Workspace, diagnostic));
```

Comment:
Ditto: why not pass the workspace through?

Reply:
➡️ Tracking as https://github.com/dotnet/roslyn/pull/43732#discussion_r416973354